GOVERNANCE.md

```diff
@@ -0,0 +1,173 @@
+- Commit to being a part of the community for the long-term.
+- Have submitted a minimum of 50 qualifying significant contributions (see list above).
+
+A Contributor is invited to become a Core Contributor by existing Core Contributors. A nomination will result in discussion and then a decision by the project steward(s).
+
+
+### Steward
+
+Steward is an additional privilege bestowed to 1 (or more) Contributors. The role of Steward is mainly an administrative one. Stewards control and maintain sensitive project assets, and act as tiebreakers in the event of disagreements. These additional privileges include:
+
+- Access to the [@astrodotbuild Twitter account](https://twitter.com/astrodotbuild)
+- Administration privileges on the [astro GitHub org](https://github.com/snowpackjs)
+- Administration privileges on the [astro Discord server](https://astro.build/chat)
+- Publish access to the [`astro` npm package](https://www.npmjs.com/package/astro)
+- Domain registrar and DNS access to `astro.build` and all other domains
+- Administration access to the `astro.build` Vercel account
+- Ability to initiate a [vote](GOVERNANCE.md#voting)
+- Ability to veto [votes](GOVERNANCE.md#voting) and resolve voting deadlocks
+- Define project direction and planning
+- Ability to decide on moderation decisions
+- Access to the `*@astro.build` email address
+
+**Nomination:**
+- Stewards cannot be self-nominated. 
+- Only Core Contributors are eligible.
+- New stewards will be added based on a unanimous vote by the existing stewards.
+- In the event that someone is unreachable then the decision will be deferred. Discussion and approval will be done in private.
+
+## Voting
+
+Certain project decisions require a vote. These include:
+
+- Governance changes: simple majority (over 50%) conducted via GitHub PR approval.
+- Core contributor membership: overwhelming majority (over 70%) conducted by privately messaging a steward. Funneling both assenting and dissenting votes directly through stewards allows for anonymity when discussing the merits of a potential contributor.
+
+A steward may initiate a vote for any unlisted project decision. Core contributors can request a vote by contacting a steward.
+
+### Rules
+
+- Members may abstain from a vote.
+- Members who do not vote within 3 days will automatically abstain.
+- Stewards may reduce the 3 day automatic abstain for urgent decisions.
+- Stewards reserve the right to veto approval with a publicly disclosed reason.
+
+
+## Moderation
+
+Outlined below is the process for Code of Conduct violation reviews.
+
+### Reporting
+
+Anyone may report a violation. Violations can be reported in the following ways:
+
+- In private, via email to one or more stewards.
+- In private, via direct message to a project steward on Discord.
+- In public, via a GitHub comment (mentioning `@rometools/stewards`).
```

Comment:
I think this needs to be changed to @snowpackjs/astro/stewards or something like that.

Reply:
lol shoot thought I got them all